pom.xml

```diff
@@ -27,7 +27,7 @@
     <parent>
         <groupId>com.google.cloud.samples</groupId>
         <artifactId>shared-configuration</artifactId>
-        <version>1.0.9</version>
+        <version>1.0.10</version>
```

Comment:
Oh not this one. 

The speech/cloud-client/pom.xml
Similar to the beta file: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/speech/beta/pom.xml#L26-L30